projects/openssh.com/package.yml

```diff
@@ -0,0 +1,85 @@
+distributable:
+  url: https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-{{version.marketing}}p1.tar.gz
+  strip-components: 1
+versions:
+  url: https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/
+  match: /openssh-\d+\.\d+p1\.tar\.gz/
+  strip:
+    - /^openssh-/
+    - /p1\.tar\.gz/
+dependencies:
+  nlnetlabs.nl/ldns: '*'
+  developers.yubico.com/libfido2: '*'
+  openssl.org: '*'
+  kerberos.org: '*'
+  thrysoee.dk/editline: '*'
+  github.com/besser82/libxcrypt: '*'
+  zlib.net: '*'
+  linux:
+    linux-pam.org: '*'
+build:
+  dependencies:
+    freedesktop.org/pkg-config: '*'
+    curl.se: '*'
+    darwin:
+      gnu.org/patch: '*'
+    linux:
+      gnu.org/gcc: '*'
+      gnu.org/make: '*'
+  script:
+    - run: |
+        curl -L "$PATCH1" | patch || true
+        curl -L "$PATCH2" | patch || true
+      if: darwin
```

Comment:
I wouldn't mask errors here. We need to know if they fail. (I know it's a pain when rebuilding without cleaning.)